src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -340,6 +340,11 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 
             THROW_IF_FAILED(ConptyResizePseudoConsole(_hPC.get(), dimensions));
             THROW_IF_FAILED(ConptyReparentPseudoConsole(_hPC.get(), reinterpret_cast<HWND>(_initialParentHwnd)));
+
+            if (_initialVisibility)
+            {
+                THROW_IF_FAILED(ConptyShowHidePseudoConsole(_hPC.get(), _initialVisibility));
+            }
```

Comment:
I'm a sucker for perf, but I think it'd be more robust to always call that function.
IMO this code is currently similar to:
```cpp
if (dimensions != COORD{ 120, 30 })
{
    THROW_IF_FAILED(ConptyResizePseudoConsole(_hPC.get(), dimensions));
}
```